units/sdlrenderer.inc

```diff
@@ -77,13 +77,18 @@ const
   SDL_TEXTUREMODULATE_COLOR = TSDL_TextureModulate(1); {**< srcC = srcC * color *}
   SDL_TEXTUREMODULATE_ALPHA = TSDL_TextureModulate(2); {**< srcA = srcA * alpha *}
 
+type
   {**
    *  Flip constants for SDL_RenderCopyEx
    *}
+  PPSDL_Flip = ^PSDL_Flip;
+  PSDL_Flip = ^TSDL_Flip;
+  TSDL_Flip = type cint;
```

Comment:
This is called `SDL_RendererFlip` in the C version.

Reply:
RIght, thanks, fixed.